contracts/FlowTransactionScheduler.cdc

```diff
@@ -1284,6 +1285,14 @@ access(all) contract FlowTransactionScheduler {
                 return
             }
 
+            // Skip processing if the epoch is in one of the phase transitions
+            // or expensive operations
+            // We don't skip if it is running in a test environment,
+            // so we check automaticRewardsEnabled() because it is only true on testnet and mainnet.
+            if FlowEpoch.isPhaseTransition() && FlowEpoch.automaticRewardsEnabled() {
```

Comment:
I want to skip this check if it is on the emulator because epochs don't run on the emulator. Is checking automatic rewards enabled an okay way to do that? I assume that it is only enabled on testnet and mainnet